examples/nvme/perf/perf.c

```diff
@@ -899,6 +899,19 @@ register_workers(void)
 	return 0;
 }
 
+static void
+unregister_worker(void)
+{
+        struct worker_thread *worker = g_workers;
```

Comment:
This seems to be using spaces for indentation; please use tabs for consistency with the rest of the codebase.